src/renderer/components/state-table.tsx

```diff
@@ -224,6 +312,191 @@ export class StateTable extends React.Component<
     );
   }
 
+  private renderExternalConfig(): JSX.Element | null {
+    if (this.state.externalData && this.state.rootStateData) {
+      const externalConfigData = this.getPoliciesAndDefaultsExternalConfig();
+      const externalConfigDefaultJSON = (
+        <JSONView
+          data={externalConfigData.default}
+          raw={'defaults: ' + externalConfigData.defaultRaw}
+          state={this.props.state}
+        />
+      );
+      const externalConfigPolicyJSON = (
+        <JSONView
+          data={externalConfigData.policies}
+          raw={'policies: ' + externalConfigData.policiesRaw}
+          state={this.props.state}
+        />
+      );
+      const rootData = this.getPoliciesAndDefaultsRoot();
+      const rootDefaultJSON = (
+        <JSONView
+          data={rootData.default}
+          raw={'defaults: ' + rootData.defaultRaw}
+          state={this.props.state}
+        />
+      );
+      const rootPoliciesJSON = (
+        <JSONView
+          data={rootData.policies}
+          raw={'policies: ' + rootData.policiesRaw}
+          state={this.props.state}
+        />
+      );
+
+      const compareDefaults = isEqual(
+        rootData.default,
+        externalConfigData.default,
+      );
+      const comparePolicies = isEqual(
+        rootData.policies,
+        externalConfigData.policies,
+      );
+
+      const message = this.getMessage(compareDefaults, comparePolicies);
+
+      return (
+        <div>
+          <Card
+            className="StateTable-Info"
+            elevation={Elevation.ONE}
+            style={{ paddingRight: '0' }}
```

Comment:
Not blocking the PR, but I still would prefer if the message content was in a box to the right of the external config comparison Card, with a 2:1 width split.